app/poros/quote.rb

```diff
@@ -0,0 +1,18 @@
+class Quote
+  attr_reader :content,
+              :author
+
+  def initialize(data)
+    @content = data[:content]
+    @author = data[:author]
+    @tags_array = data[:tags]
+  end
+
+  def tags
+    if @tags_array.nil? || @tags_array.empty?
+      "no tags present"
+    else
+      @tags_array.join(", ")
+    end
+  end
+end
```

Comment:
Nice job here! I like the method `tags` that you have.

Reply:
It would also be fine to have the logic that you have in `tags` in the view itself since it could be categorized under presentation logic.